README.md

```diff
@@ -63,7 +63,7 @@ Each Lemmy server can set its own moderation policy; appointing site-wide admins
 - Lead singer from [Motörhead](https://invidio.us/watch?v=pWB5JZRGl0U).
 - The old school [video game](<https://en.wikipedia.org/wiki/Lemmings_(video_game)>).
 - The [Koopa from Super Mario](https://www.mariowiki.com/Lemmy_Koopa).
-- The [furry rodents](http://sunchild.fpwc.org/lemming-the-little-giant-of-the-north/).
+- The [furry rodents](https://en.wikipedia.org/wiki/Lemming).
```

Comment:
There is nothing wrong with this, a readonly page doesnt need https. And wikipedia is boring.

Reply:
fair enough - considering this is a git readme it's probably not for general users.

as a note, i feel that having links with security errors on public facing sites can definitely scare away new users however!